vendor/github.com/moby/buildkit/vendor.conf

```diff
@@ -4,10 +4,10 @@ github.com/pkg/errors v0.8.0
 github.com/stretchr/testify v1.1.4
 github.com/davecgh/go-spew v1.1.0
 github.com/pmezard/go-difflib v1.0.0
-golang.org/x/sys 314a259e304ff91bd6985da2a7149bbf91237993
+golang.org/x/sys 1b2967e3c290b7c545b3db0deeda16e9be4f98a2
```

Comment:
✅ docker/cli is already ahead; https://github.com/golang/sys/compare/ac767d655b305d4e9612f5f6e33120b9176c4ad4...1b2967e3c290b7c545b3db0deeda16e9be4f98a2